Remotion/Web/Core/UI/HtmlHeadAppenderExtensions.cs

```diff
@@ -31,13 +31,24 @@ public static class HtmlHeadAppenderExtensions
     /// <summary>
     /// Registers Utilities.js in non-themed HTML folder of Remotion.Web.dll.
     /// </summary>
+    public static void RegisterWebClientScript (this HtmlHeadAppender htmlHeadAppender)
+    {
+      ArgumentUtility.CheckNotNull("htmlHeadAppender", htmlHeadAppender);
+
+      string webClientScriptKey = typeof(HtmlHeadContents).GetFullNameChecked() + "_Web_ClientScript";
+      var webClientScriptUrl = ResourceUrlFactory.CreateResourceUrl(typeof(HtmlHeadContents), ResourceType.Html, "Web.ClientScript.js");
```

Comment:
Performance optimization:
since we now register the same script many times over, we should first test `htmlHeadAppender.IsRegistered(key) before resolving the url factory and building the url.

Goes for the other HtmlheadAppenderExtension, too.

As a improvement, we could make the key "Remotion.Web.ClientScript" (and "Remotion.ObjectBinding.Web.ClientScript") instead of building it via the resolved name. Those where just done for cention and to make things easier but it's not a good idea when it comes to performance.